mxcube3/routes/signals.py

```diff
@@ -384,9 +388,9 @@ def collect_oscillation_failed(
 
     if not qutils.is_interleaved(node["node"]):
         try:
-            limsres = blcontrol.rest_lims.get_dc(lims_id)
+            blcontrol.rest_lims.get_dc(lims_id)
         except BaseException:
-            limsres = {}
+            pass
```

Comment:
this try bloc is not needed

Reply:
The call to get_dc can still go wrong so the try block would still be needed, unless you see a scenario when this never will go wrong ?

Reply:
sorry, I missed this comment. I meant the actual call is not needed in this method